modules/op-specifying-pvc-as-volumesource-in-workspaces.adoc

```diff
@@ -5,14 +5,15 @@
 [id="specifying-pvc-as-volumesource-in-workspaces_{context}"]
 = Specifying PersistentVolumeClaims as VolumeSource in Workspaces
 
-Workspaces help Tasks share data, and allow you to specify one or more volumes that each Task in the Pipeline requires during execution.
+Workspaces help tasks share data, and allow you to specify one or more volumes that each task in the pipeline requires during execution.
+You can either create a persistent volume claim or provide a volume claim template, which creates a persistent volume claim for you.
 
-In this section, you will create a PersistentVolumeClaim to provide data storage and bind it to the Workspace. This PersistentVolumeClaim provides the volumes or filesystem required for the Pipeline execution.
+In this section, you will create a persistent volume claim to provide data storage and bind it to the workspace. This persistent volume claim provides the volumes or filesystem required for the pipeline execution.
```

Comment:
Since we have replaced `PersistentVolumeClaim` with `VolumeClaimTemplate` do we still need the steps to create `PersistentVolumeClaim`?

Reply:
@VeereshAradhya I have removed this module and added the relevant conceptual part to the Workspaces concept docs.